metricbeat/track.json

```diff
@@ -0,0 +1,30 @@
+{% import "rally.helpers" as rally with context %}
+{
+  "version": 2,
+  "description": "Metricbeat data",
+  "indices": [
+    {
+      "name": "metricbeat",
+      "body": "index.json"
+    }
+  ],
+  "corpora": [
+    {
+      "name": "metricbeat",
```

Comment:
can you please add the following property:

```json
"base-url": "http://benchmarks.elasticsearch.org.s3.amazonaws.com/corpora/metricbeat"
```

Reply:
I pushed b9326